src/seedu/addressbook/ui/TextUi.java

```diff
@@ -26,12 +26,6 @@
 
     private static final String DIVIDER = "===================================================";
```

Comment:
hmmmm....depending on how you define the responsibility of a Formatter, maybe this could be moved to the `Formatter` class too?